packages/daemon/src/daemon.js

```diff
@@ -404,6 +396,82 @@ const makeEndoBootstrap = async (
           }))(),
         internal: undefined,
       };
+    } else if (formula.type === 'handle') {
+      context.thisDiesIfThatDies(formula.target);
+      return {
+        external: {},
+        internal: {
+          targetFormulaIdentifier: formula.target,
+        },
+      };
+    } else if (formula.type === 'endo') {
+      /** @type {import('./types.js').EndoBootstrap} */
+      const endoBootstrap = Far('Endo private facet', {
+        // TODO for user named
```

Comment:
Qué?

Reply:
idk it was already there

Reply:
The comment can be deleted. It’s a note to myself to consider adding the user’s name to the string above in the first argument of `Far` but that itself raises the question: “what name?”. It could be the first name assigned, but that raises the question: “by whom?”. The first agent to give them a name? Not sure it’s a good idea anyway.